cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -65,11 +67,13 @@ def full_test_vm(setup_provider_modscope, provider, full_template_modscope, requ
         logger.exception('Exception deleting test vm "%s" on %s', vm.name, provider.name)
 
 
-def new_snapshot(test_vm, has_name=True, memory=False):
+def new_snapshot(test_vm, has_name=True, memory=False, description=True):
```

Comment:
This was added so we can create a new snapshot with missing description.

Reply:
**REQUIRED**: The naming of this gives the impression that it sets the description, it should be changed to be ```create_description```, which then makes sense with a bool value.